Description:Include missing lib to avoid build failure on 
deepin-riscv.
Author:Gui-Yue <yuemeng.gui@gmail.com>
--- a/src/lyxfind.cpp
+++ b/src/lyxfind.cpp
@@ -13,6 +13,7 @@
  */
 
 #include <config.h>
+#include <iterator>
 
 #include "lyxfind.h"
 
--- a/src/insets/InsetListings.cpp
+++ b/src/insets/InsetListings.cpp
@@ -10,7 +10,7 @@
  */
 
 #include <config.h>
-
+#include <cstring>
 #include "InsetListings.h"
 
 #include "Buffer.h"
